src/components/TutorDashboard/index.js

```diff
@@ -150,6 +150,8 @@ function Tutor(props) {
   const [endTime, setendTime] = useState("");
   const [slicedStudents, setSlicedStudents] = useState([]);
   const [statusValue, setStatusValue] = useState("");
+  const [pathwayClicked, setPathwayClicked] = useState(true);
+  const [pathwayTwoClicked, setPathwayTwoClicked] = useState(false);
```

Comment:
You want to avoid having dependent state like this where one is always the opposite of the other because (a) we have to remember to update both together (or create a function that does this and remember to call that function) and (b) given we're currently using React < 18, this could unnecessarily cause extra re-renders (See: https://dev.to/this-is-learning/automatic-batching-in-react-18-273h#:~:text=Batching%20is%20when%20React%20groups,batched%20in%20React%20by%20default.)

```suggestion
  const [pathwayClicked, setPathwayClicked] = useState(true);
  const pathwayTwoClicked = !setPathwayClicked;
```